apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data/FirestoreDb.cs

```diff
@@ -148,5 +153,44 @@ internal DocumentReference GetDocumentReferenceFromResourceName(string reference
                 nameof(referenceValue), "Reference {0} is not part of database {1}", referenceValue, RootPath);
             return Document(referenceValue.Substring(DocumentsPath.Length + 1));
         }
+
+        /// <summary>
+        /// Fetches document snapshots from the server, based on an optional transaction ID.
+        /// </summary>
+        /// <param name="documents">The document references to fetch. Must not be null.</param>
```

Comment:
Looks like it can't contain null element either. Is that worth mentioning?

Reply:
Yup, that would be a good idea. Will do, and will make it throw an ArgumentException accordingly.